storybook/webpack.config.js

```diff
@@ -4,15 +4,11 @@ const isCi = process.env.CI && process.env.CI !== '';
 
 module.exports = async ({ config }) => {
   const [jsxRule] = config.module.rules;
-  jsxRule.use.unshift('thread-loader');
+  // jsxRule.use.unshift('thread-loader');
+
   // Use Auto DLL plugin for faster development builds
+
   if (!isCi) {
-    const [htmlWebpackPlugin] = config.plugins;
```

Comment:
Moved to https://github.com/input-output-hk/daedalus/pull/2760/files#diff-3f1f6f46f397331169d53cb931c669f53ef20add9879cd6bde6955dd7ff6cdf1R14